Controllers/AppointmentController.cs

```diff
@@ -94,9 +92,9 @@ public async Task<IActionResult> GetAppointment(int appointmentId)
             }
         }
 
-        [HttpGet("get-all")]
+        [HttpGet("appointments")]
```

Comment:
here endpoint will be `appointment/appointments` which is not good. So lets keep it `appointment/all`